apps/openmw/mwscript/miscextensions.cpp

```diff
@@ -923,11 +923,7 @@ namespace MWScript
                 if (!ptr.isEmpty())
                 {
                     const std::string& script = ptr.getClass().getScript(ptr);
-                    if (script.empty())
-                    {
-                        output << ptr.getCellRef().getRefId() << " has no script " << std::endl;
```

Comment:
How do a user see a difference is it a global variable or a script variable?

Reply:
I changed the format to the way vanilla does it with slight deviations (a member separator, dot instead of ->, the case of the original variable name isn't preserved). Hopefully it's clear enough.